Source/ProgressionSystemRuntime/Private/LevelActors/PSStarActor.cpp

```diff
@@ -86,3 +128,38 @@ void APSStarActor::TryPlayHideStarAnimation()
 		StartTimeHideStarsInternal = 0.f;
 	}
 }
+
+// Menu stars with animation 
+void APSStarActor::TryPlayMenuStarAnimation(float DeltaTime)
+{
+	const FPSRowData& CurrentRow = UPSWorldSubsystem::Get().GetCurrentRow();
+	if (!CurrentRow.MenuStarsAnimation)
+	{
+		SetActorTickEnabled(false);
+		return;
+	}
+
+	FCurveTableRowHandle Handle;
+	Handle.CurveTable = CurrentRow.MenuStarsAnimation;
+	Handle.RowName = FName("RotationYaw");
+
+	const FRealCurve* Curve = CurrentRow.MenuStarsAnimation ? Handle.CurveTable->FindCurve(Handle.RowName, TEXT("Rotation")) : nullptr;
```

Comment:
you already use `UGameplayUtilsLibrary::ApplyTransformFromCurveTable`, you don't need `MaxTime` as it will return `false` for the last value. See TryPlayHideStarAnimation function

Reply:
done as part of https://github.com/h4rdmol/ProgressionSystem/commit/a5231652e33f7f150dc6d860ded531e0a8493cb3